web/modules/weather_data/src/Service/Test/GetFromAPI.php.test

```diff
@@ -33,25 +33,22 @@ final class GetFromApiTest extends Base
      */
     public function testCachesApiData(): void
     {
+        $expected = "cached value";
+        $this->cacheMock
+            ->method("get")
+            ->willReturn((object) ["data" => $expected]);
+
+        // Mock an API response. This should not be called.
         $this->httpClientMock->append(
             new Response(
                 200,
                 ["Content-type" => "application/json"],
                 '{"response":"here"}',
             ),
         );
-        $first = $this->weatherDataService->getFromWeatherAPI("my url");
 
-        // Change the response and call it again. We should get the same response.
-        $this->httpClientMock->append(
-            new Response(
-                200,
-                ["Content-type" => "application/json"],
-                '{"here":"response"}',
-            ),
-        );
-        $second = $this->weatherDataService->getFromWeatherAPI("my url");
```

Comment:
Since we're mocking the cache now, we no longer need to actually make this call twice. :yay: